src/herder/test/HerderTests.cpp

```diff
@@ -3023,6 +3023,134 @@ TEST_CASE("SCP checkpoint", "[catchup][herder]")
     }
 }
 
+// This test confirms that tx set processing and consensus are independent of
+// the tx queue source account limit (for now)
+TEST_CASE("tx queue source account limit", "[herder][transactionqueue]")
+{
+    std::shared_ptr<Simulation> simulation;
+    std::shared_ptr<Application> app;
+    std::shared_ptr<Application> limitApp;
+
+    auto setup = [&](bool mix) {
+        auto networkID = sha256(getTestConfig().NETWORK_PASSPHRASE);
+        simulation = std::make_shared<Simulation>(
+            Simulation::OVER_LOOPBACK, networkID, [mix](int i) {
+                auto cfg = getTestConfig(i, Config::TESTDB_ON_DISK_SQLITE);
+                if (!mix || i % 2 == 1)
```

Comment:
I think for additional coverage it might be worth covering mixed scenarios where the nodes with new setting is both majority and minority. Or do we have a consensus threshold of 1 here, so that wouldn't matter?

Reply:
I believe this shouldn't matter because of the threshold, which requires all nodes to vote in order to confirm a tx set (which means all nodes must consider the value "valid")